tests/hubstorage/test_frontier.py

```diff
@@ -3,126 +3,125 @@
 """
 import pytest
 
-from .conftest import TEST_FRONTIER_NAME, TEST_FRONTIER_SLOT
+from .conftest import TEST_FRONTIER_SLOT
 
 
 @pytest.fixture(autouse=True)
-def delete_frontier_slot(hsproject):
+def delete_frontier_slot(hsproject, frontier_name):
     frontier = hsproject.frontier
-    frontier.delete_slot(TEST_FRONTIER_NAME, TEST_FRONTIER_SLOT)
+    frontier.delete_slot(frontier_name, TEST_FRONTIER_SLOT)
 
 
 def _get_urls(batch):
     return [r[0] for r in batch['requests']]
 
 
-def test_add_read(hsproject):
+def test_add_read(hsproject, frontier_name):
     frontier = hsproject.frontier
 
     fps = [{'fp': '/'}]
-    frontier.add(TEST_FRONTIER_NAME, TEST_FRONTIER_SLOT, fps)
+    frontier.add(frontier_name, TEST_FRONTIER_SLOT, fps)
     fps = [{'fp': '/index.html'}, {'fp': '/index2.html'}]
-    frontier.add(TEST_FRONTIER_NAME, TEST_FRONTIER_SLOT, fps)
+    frontier.add(frontier_name, TEST_FRONTIER_SLOT, fps)
     frontier.flush()
 
     urls = [_get_urls(batch) for batch
-            in frontier.read(TEST_FRONTIER_NAME, TEST_FRONTIER_SLOT)]
+            in frontier.read(frontier_name, TEST_FRONTIER_SLOT)]
     expected_urls = [[u'/', u'/index.html', u'/index2.html']]
     assert urls == expected_urls
 
 
-def test_add_multiple_chunks(hsproject):
+def test_add_multiple_chunks(hsproject, frontier_name):
     frontier = hsproject.frontier
     old_count = frontier.newcount
 
     batch_size = 50
     fps1 = [{'fp': '/index_%s.html' % fp} for fp in range(0, batch_size)]
-    frontier.add(TEST_FRONTIER_NAME, TEST_FRONTIER_SLOT, fps1)
+    frontier.add(frontier_name, TEST_FRONTIER_SLOT, fps1)
 
     fps2 = [{'fp': '/index_%s.html' % fp}
             for fp in range(batch_size, batch_size * 2)]
-    frontier.add(TEST_FRONTIER_NAME, TEST_FRONTIER_SLOT, fps2)
+    frontier.add(frontier_name, TEST_FRONTIER_SLOT, fps2)
 
     fps3 = [{'fp': '/index_%s.html' % fp}
             for fp in range(batch_size * 2, batch_size * 3)]
-    frontier.add(TEST_FRONTIER_NAME, TEST_FRONTIER_SLOT, fps3)
+    frontier.add(frontier_name, TEST_FRONTIER_SLOT, fps3)
     frontier.flush()
 
     assert frontier.newcount == 150 + old_count
 
     # insert repeated fingerprints
-    fps4 = [{'fp': '/index_%s.html' % fp} for fp in range(0, batch_size)]
```

Comment:
@immerrr last question about the PR before merge, why the test with repeated fps was changed?

Reply:
Because `fps4` was unused and `fps3` were used as "repeated fingerprints" instead.

Reply:
Oh, thanks for cleaning it, I didn't spot it even now :man_facepalming: